### PR TITLE
Fixed manage trip endpoint

### PR DESCRIPTION
**Pull request by Grasmus**

### CODE REVIEW

**Review by Eugene-pas — COMMENTED** [review]
The function uses the change "manageTrip.OffersId" (file TripService.cs) with fields that do not exist. This generates a fatal error.
image

File: Core/Specifications/OfferSpecification.cs [hunk 1]
```diff
@@ -5,6 +5,7 @@
 using NetTopologySuite.Geometries;
 using System;
 using System.Collections.Generic;
+using System.Linq;
```
Comment (Eugene-pas) [inline Core/Specifications/OfferSpecification.cs:7]: delete extra usings

File: Core/Specifications/OfferSpecification.cs [hunk 2]
```diff
@@ -66,10 +70,10 @@ internal class GetOffersNearRoute : Specification<Offer>
 
         internal class GetOfferByIds : Specification<Offer>
         {
-            public GetOfferByIds(List<int> offers)
+            public GetOfferByIds(List<int> offersIds)
             {
                 Query
-                    .Where(offer => offers.Contains(offer.Id));
+                    .Where(offer => offersIds.Contains(offer.Id));
```
Comment (Eugene-pas) [inline Core/Specifications/OfferSpecification.cs]: You talked about the style of writing specifications, rewrite everything abbreviated.

File: Core/Services/TripService.cs [hunk 3]
```diff
@@ -169,33 +168,48 @@ public async Task ManageOffersTripAsync(ManageTripDTO manageTrip)
...
 
-                ExceptionMethods.PointNullCheck(pointData);
+                    ExceptionMethods.PointNullCheck(pointData);
 
-                points.Add(pointData);
+                    pointData.Address = point.Address;
+                    pointData.Settlement = point.Settlement;
+                    pointData.Country = point.Country;
+                    pointData.Postcode = point.Postcode;
+                    pointData.Region = point.Region;
```
Comment (Eugene-pas) [inline Core/Services/TripService.cs]: This code is not required.
`pointData.Address = point.Address;
                    pointData.Settlement = point.Settlement;
                    pointData.Country = point.Country;
                    pointData.Postcode = point.Postcode;
                    pointData.Region = point.Region;`

**Review by AndriiChepeliuk — APPROVED** [review]

**Review by Eugene-pas — APPROVED** [review]

**Pull request merged**